build/linux/dist/arduino

```diff
@@ -27,7 +27,7 @@ fi
 # Collect options to java in an array, to properly handle whitespace in options
 JAVA_OPTIONS=("-DAPP_DIR=$APPDIR" "-Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel")
 
-if [[ "$@" != *"--upload"* && "$@" != *"--verify"* && "$@" != *"--get-pref"* && "$@" != *"--pref"* && "$@" != *"--install-board"* && "$@" != *"--install-library"* ]] ; then
+if [[ "$@" != *"--upload"* && "$@" != *"--verify"* && "$@" != *"--get-pref"* && "$@" != *"--save-prefs"* "$@" != *"--install-boards"* && "$@" != *"--install-library"* ]] ; then
```

Comment:
> Maybe this does not cover all the use-cases, 

I wonder if it would be better to just suppress the splash screen whenever *any* options are passed, since keeping this list of options up-to-date is probably going to fail (again) in the future. The most-used invocation is without options, which should show a test screen. The invocations with options that do show the GUI and should (ideally) show a splash screen too are probably very seldomly used, so it might be acceptable to not show a splash screen in those cases.
> Somehow I was hoping that this would fix the X11 DISPLAY requirement (as the original error message was related to the Splash-screen), but that's rooted much deeper into the code I learned by experience :-/ So this is a minor improvement only.

Ah, I already wondered about that indeed :-)